front-end/src/component/Authentification/PassangerProfile.js

```diff
@@ -0,0 +1,75 @@
+import React, { Commit } from "react";
+import { Link } from "react-router-dom";
+
+class PassangerProfile extends Commit() {
```

Comment:
Extends Commit ? Hey what is going on here @veemish ?

Reply:
oh my that's a typo

Reply:
So should I also change it to a functional component?

Reply:
stateless i mean